Kitodo/src/test/java/org/kitodo/selenium/ListingST.java

```diff
@@ -234,4 +235,27 @@ public void listUsersTest() throws Exception {
         int ldapGroupsDisplayed = usersPage.countListedLdapGroups();
         assertEquals("Displayed wrong number of ldap groups!", ldapGroupsInDatabase, ldapGroupsDisplayed);
     }
+
+    /**
+     * Test number of displayed templates. Assert deactivated templates are hidden by default and only visible
+     * when activating the corresponding switch on the projects/templates page.
+     *
+     * @throws Exception when thread is interrupted or templates cannot be loaded.
+     */
+    @Test
+    public void listTemplatesTest() throws Exception {
+        projectsPage.goToTemplateTab();
+        assertEquals("Wrong number of templates before hiding first template", 2,
+                projectsPage.getTemplateTitles().size());
+        TemplateEditPage editTemplatePage = projectsPage.editTemplate();
+        editTemplatePage.hideTemplate();
+        editTemplatePage.save();
+        assertEquals("Wrong number of templates after hiding first template", 1,
+                projectsPage.getTemplateTitles().size());
+        projectsPage.goToTemplateTab();
+        Thread.sleep(500);
```

Comment:
Considering that the CI pipeline already needs 20+ minutes to run, maybe it is possible to avoid generic sleep statements (I know there are a lot of others). There are some examples in other Selenium tests on how to wait for certain conditions like html-elements being available.